go/pserver/service.go

```diff
@@ -52,14 +67,34 @@ type Service struct {
 	optMap map[string]*optimizer
 }
 
+type checkpoint struct {
+	Uuid      string
```

Comment:
Adding `json: "uuid"` at the end of the line,  so we can use `Json.marshal` to a format JSON.